adlfs/spec.py

```diff
@@ -1233,9 +1233,12 @@ async def _rm_files(
 
             # Files and directory markers of empty directories can be deleted in any order. We delete them all
             # asynchronously for performance reasons.
-            file_exs = await asyncio.gather(
-                *([cc.delete_blob(file) for file in files]), return_exceptions=True
-            )
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+
+                file_exs = await asyncio.gather(
+                    *([cc.delete_blob(file) for file in files]), return_exceptions=True
+                )
```

Comment:
If there is another `warnings.simplefilter("error")` deeply hidden inside `cc.delete_blob`, our `simplefilter("ignore")` will possibly be overridden. This could possibly be the reason why it did not fix the pipeline.